ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```
@@ -194,8 +362,23 @@ function is_dualstack() {
   fi
 }
 
-trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
-trap 'prepare_next_steps' EXIT TERM
+function cleanup() {
+  local exit_code=$?
+
+  # Kill any background jobs
+  CHILDREN=$(jobs -p)
+  if test -n "${CHILDREN}"; then
+    kill ${CHILDREN} 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)